File: Sem-3/advance_c_programming/31.c
#include <stdio.h>
void main()
{
	int a[100],i,n;
	int (*p);
	printf("enter the number of element:\n");
	scanf("%d",&n);
	printf("enter %d element here:\n",n);
	for(i=0;i<n;i++)
	{
		scanf("%d",&a[i]);
	}
	p=&a;
	printf("array elements are :\n");
	for(i=0;i<n;i++)
	{
		printf("%d\n",*(p+i));
	}
	printf("value of pointer is :%d\n",p);
}

File: Sem-4/Python/practical exam/31.py
n=int(input("enter size of set"))
a=set()
for i in range(n):
    x=int(input("Enter element"))
    a.add(x)
    
n1=int(input("enter size of set"))
b=set()
for i in range(n1):
    y=int(input("Enter element"))
    b.add(y)
    
print(a.union(b))
